pallets/subtensor/rpc/src/lib.rs

```diff
@@ -41,7 +41,13 @@ pub trait SubtensorCustomApi<BlockHash> {
     fn get_neurons(&self, netuid: u16, at: Option<BlockHash>) -> RpcResult<Vec<u8>>;
     #[method(name = "neuronInfo_getNeuron")]
     fn get_neuron(&self, netuid: u16, uid: u16, at: Option<BlockHash>) -> RpcResult<Vec<u8>>;
-
+    #[method(name = "neuronInfo_getNeuronCertificate")]
```

Comment:
We are deprecating the addition of new RPCs. going forwards ,  clients should make query state directly , for what they nee. That being said , since this is a part of the Axon , it can be fetched from neuron info which the cli already uses to populate the metagraph

Reply:
[It's not really part of the Axon](https://github.com/opentensor/subtensor/pull/750/files#diff-9d85ff26db441ec381d86188b389b226449eee4e9847bf0ce3301c0bb6cf93a0R99) - it's a separate storage. I thought that adding it to the aoxn  (can't use any the placeholders due to the type) would break the encoding/decoding of the AxonInfo on subtensor update - right?

Reply:
Got it , leaning towards seperate methods. Also we could introduce an Axonv2 , if we dont want to break existing integration. 

 The recommendation to remove rpcs still hold. Client should access this data via [direct state queries](https://gist.github.com/JohnReedV/de955830d499979e80da212ab03fa7b7)

Reply:
removed the rpc